### PR TITLE
Webadmin added 

### PR DESCRIPTION
**Pull request by Exellent1988**
Webadmin um die neuen Nummern einzupflegen in die Que und die Sitzplätze frei und belegt zu schalten.

### CODE REVIEW

File: Webadmin/main.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
+
+        // Globally initializes an mqtt variable
+        const clientId = 'webadmin_' + Math.random().toString(16).substr(2, 8)
+currentNumber = 1
+const host = 'ws://ztl@ztl-mqtt.westeurope.azurecontainer.io:15675/ws'
+var serverUrl   = 'ws://ztl@ztl-mqtt.westeurope.azurecontainer.io:15675/ws';     /* wss://mqtt.cumulocity.com/mqtt for a secure connection */
+var device_name = "Webadmin";
+var username    = "ztl";
+var password    = "Ztl2019!";
```
Comment (cboltz) [inline Webadmin/main.js:9]: Shouldn't that be `topsecret`? ;-)

(in related news: you want to change that password)

File: Webadmin/main.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+ $("#table-"+tablenum+'> span').addClass('bg-info');
+}
+function TableEnd(tablenum){
+  message = {
+  "place_number": tablenum,
+  "place_occupied": false
+}
+ publish('macherdaach/queue/messageFromPlace',JSON.stringify(message));
+$("#table-"+tablenum+'> span').removeClass('bg-info');
+}
```
Comment (cboltz) [inline Webadmin/main.js:108]: This section needs some whitespace fixes to make the code readable.

Comment (cboltz) [reply Webadmin/main.js:108]: Hmm, why does github allow to mark multiple lines, and then only shows a few? :-/

With "This section", I mean lines 75 to 108 in main.js.

Comment (Exellent1988) [reply Webadmin/main.js:108]: Who ever needs readable code can use a beautifyer ;)

File: Webadmin/main.js (hunk 1 above)
Comment (Exellent1988) [reply Webadmin/main.js:9]: Yes but that's the way the quick and dirty push, it had to be done in a manner of short timing schedule.

**Pull request merged**